hmac/decryptor/hashQuery_test.go

```diff
@@ -204,3 +204,99 @@ func TestSearchableWithTextFormat(t *testing.T) {
 		assert.Equal(t, len(rightVal.Val), len(newData))
 	}
 }
+
+// TestSearchableWithJoinsWithTextFormat process searchable SELECT query with placeholder for prepared statement
+// and use binding values in text format
+func TestSearchableWithJoinsWithTextFormat(t *testing.T) {
+	clientSession := &mocks.ClientSession{}
+	sessionData := make(map[string]interface{}, 2)
+	clientSession.On("GetData", mock.Anything).Return(func(key string) interface{} {
+		return sessionData[key]
+	}, func(key string) bool {
+		_, ok := sessionData[key]
+		return ok
+	})
+	clientSession.On("DeleteData", mock.Anything).Run(func(args mock.Arguments) {
+		delete(sessionData, args[0].(string))
+	})
+	clientSession.On("SetData", mock.Anything, mock.Anything).Run(func(args mock.Arguments) {
+		sessionData[args[0].(string)] = args[1]
+	})
+
+	schemaConfig := `schemas:
+  - table: test_table
+    columns:
+      - data1
+    encrypted:
+      - column: data1
+        searchable: true
+
+  - table: test_table_2
+    columns:
+      - data1
+    encrypted:
+      - column: data1
+        searchable: true
+`
+
+	type testcase struct {
+		Query string
+	}
+
+	testcases := []testcase{
+		{Query: "SELECT * FROM table1 t1 inner join test_table t2 inner join test_table_2 t3 on t2.data1=t3.data1"},
```

Comment:
can you add cases with tables without aliases? join between tables with alias and without? with full qualifiers with table names like `test_table.data1 = t2.tdata1`? to cover different cases